articles/cognitive-services/form-recognizer/quickstarts/python-train-extract.md

```diff
@@ -24,7 +24,7 @@ If you don't have an Azure subscription, create a [free account](https://azure.m
 -  You must get access to the Form Recognizer limited-access preview. To get access to the preview, please fill out and submit the [Cognitive Services Form Recognizer access request](https://aka.ms/FormRecognizerRequestAccess) form. 
 - You must have [Python](https://www.python.org/downloads/) installed if you want to run the sample locally.
 - You must have a subscription key for Form Recognizer. To get a subscription key, see [Obtaining Subscription Keys](https://docs.microsoft.com/azure/cognitive-services/cognitive-services-apis-create-account).
-- You must have a minimum set of five forms of the same type. You can use a [sample dataset](https://go.microsoft.com/fwlink/?linkid=2090451) for this quickstart.
+- You must have a minimum set of five forms of the same type. You can use a [sample dataset](https://go.microsoft.com/fwlink/?linkid=2090451) for this quickstart. Note - sample data (training data) should be on a blob without subfolders. 
```

Comment:
instead of the note, let's say "Upload the data to the root of an Azure Blob Storage account. You will use this data to train the model."